app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,330 @@
+  /*font-family: 'Helvetica', sans-serif;*/
+  /*font-family: 'Quicksand', sans-serif;*/
+  /*font-family: 'Roboto', sans-serif;*/
+  font-family: 'Montserrat', sans-serif;
+  /*font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif*/
+  margin: 20px;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: 0px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+h2 {
+  margin-bottom: 20px;
+
+}
+
+h3 {
+  font-size: 2em;
+}
+
+h3, h5 {
+  text-align: center;
+  padding-bottom: 40px;
+  padding-top: 20px;
+}
+
+footer {
+  width: 100%;
+  min-height: 30px;
+  margin: 50px auto;
+  padding: 20px 20px 40px 20px;
+  text-align: center;
+}
+
+@keyframes fadein {
+  from {
+    opacity:0;
+  }
+  to {
+    opacity:1;
+  }
+}
+
+img {
+  border-radius: 5px;
+  margin-bottom: 10px;
+}
+
+img:hover {
+  opacity: 0.8;
+}
+
+.shop-name {
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.title {
+  letter-spacing: 3px;
+  text-align: center;
+  color: black;
+  font-weight: 500;
+  margin-bottom: 50px;
+}
+
+.dropdown.menu {
+  margin-bottom: 2rem;
+  border: none;
+}
+
+.dropdown.menu a {
+  padding-left: 0rem;
+  margin-right: 2rem;
+}
+
+.dropdown.menu li {
+  padding-left: 10px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(186px, 1fr));
+  /*grid-template-columns: 1fr 1fr 1fr 1fr (auto-fit, 186px);*/
+  /*grid-template-columns: repeat(auto-fit, 186px);*/
+  grid-column-gap: 25px;
+  grid-row-gap: 25px;
+}
+
+.grid-item {
+  justify-self: center;
+  align-items: center;
+}
+
+.button {
+  background-color: #000000 !important;
+  color: #FFFFFF;
+  border-radius: 5px;
+}
+
+.merchant-options {
+  text-align: center;
+}
+
+.button:hover {
+  color: #24AB82;
+}
+
+.merchant-options {
+  text-align: center;
+}
+
+a:hover {
+  color: #24AB82;
+}
+
+
+.top-nav {
+  display: grid;
+  grid-template-columns: 8fr 1fr 1fr 1fr;
+}
+
+.cart-bottom {
+  display: grid;
+  grid-template-columns: 6fr 1fr 1fr;
+  text-align: right;
+  padding-top: 10px;
+  padding-right: 20px;
+  padding-left: 10px;
+
+}
+
+.us-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(186px, 1fr));
+  /*grid-template: 1fr 1fr / 1fr 1fr;*/
+  margin: 30px;
+  grid-column-gap: 100px;
+  grid-row-gap: 25px;
+}
+
+.us-box {
+  display: inline-block;
+  max-width: 400px;
+  min-width: 200px;
+  padding-bottom: 70px;
+}
+
+
+#footer {
+  border-top: solid thin;
+  padding-bottom: 5px;
+}
+
+#find-order {
+  text-align: left;
+}
+
+.shop-name {
+  letter-spacing: 2px;
+  text-align: center;
+}
+
+.account-page {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#ul_top_hypers li {
+  display: inline;
+}
+
+table {
+  margin-left:auto;
+  margin-right:auto;
+  border-spacing: 2px;
+  border-collapse: separate;
+}
+
+.order-fulfillment {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+}
+
+#ul_table li {
```

Comment:
I think `ul_table` is an interesting name for an ID: it doesn't really describe what's going on, besides that it's a table inside of a UL (then why wouldn't you use `ul table`?), so it doesn't really help other developers who are looking at this.

Also in general, CSS ids/classes are hyphenated